Tools/CISupport/ews-build/steps.py

```diff
@@ -7213,31 +7282,38 @@ def write_unexpected_results_file_to_master(self):
             results_data_obj = json.dumps(self.unexpected_results_filtered, indent=4)
             f.write(results_data_obj)
 
-    def decode_results_data(self, string):
+    @defer.inlineCallbacks
```

Comment:
Why is this async? I don't see anything we're calling that forces it to be.

Reply:
I'm pretty sure that calling `yield self._addToLog` requires the function to be async. I remember I fixed this because I ran into an exception but I can't find the exact build.